terraform/modules/open_id_connect/main.tf

```diff
@@ -0,0 +1,12 @@
+data "aws_region" "current" {}
+data "aws_caller_identity" "current" {}
+
+resource "aws_iam_openid_connect_provider" "github" {
+  url = "https://token.actions.githubusercontent.com"
+
+  client_id_list = [
+    "sts.amazonaws.com",
+  ]
+
+  thumbprint_list = ["6938fd4d98bab03faadb97b34396831e3780aea1"]
```

Comment:
Does GitHub Actions only have one intermediate SSL cert at the moment? Can you put a comment with where this came from in case we need to update it?

Reply:
Added comment on thumbprint source